backend/sample_env

```diff
@@ -1,11 +1,6 @@
 SECRET_KEY=yl2w)c0boi_ma-1v5)935^2#&m*r!1s9z9^*9e5co^08_ixzo6
-DATABASE_NAME=ai
-DATABASE_USER=admin
-DATABASE_PASSWORD=admin
-DATABASE_HOST=localhost
-DATABASE_PORT=5432
-DATABASE_SCHEMA=public
-GALAXY_URL=http://44.203.33.53:8000/raw-data/current-snapshot/
+export DATABASE_URL=postgis://admin:password@localhost:5432/ai
```

Comment:
oh right , No ! This was to indicate , that we can export the database url to working env without specifying on the sample_env , so that project can pick from the exported env ( for eg on docker containers we can start containers with the env predetermined )  You can remove it and should work as fine , I will remove this ```export``` on working PR !